magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt

```diff
@@ -0,0 +1,25 @@
+package link.magic.android.core.relayer
+
+import android.app.Dialog
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import android.webkit.WebView
+import android.widget.RelativeLayout
+
+class WebViewDialog(context: Context, private val webView: WebView) : Dialog(context){
```

Comment:
Can this context be ApplicationContext? Or it has to be an Activity context?

Reply:
I believe it can accept either. In this case, I'm making sure to pass it whichever context is passed by `runOnUiThread`